### PR TITLE
[T3A2][W11-A2]Fan Weiguang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -32,5 +32,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+	/**
+	 * @return the feedback message to be shown to the user, which contains a
+	 *         description of the execution result
+	 */
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: consider removing auto gen comments when they add no value. generally accessors don't need comments as their names are often self-explanatory.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: goof catch!

**Comment by cmkumar87** [comment]
@fanwgwg  Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by fanwgwg** [comment]
Thanks for commenting!

**Pull request closed without merging**